third_party/move/evm/move-to-yul/Cargo.toml

```diff
@@ -26,6 +17,12 @@ ethnum = { workspace = true }
 evm = { workspace = true }
 itertools = { workspace = true }
 maplit = { workspace = true }
+move-command-line-common = { workspace = true }
+move-compiler = { workspace = true }
+move-core-types = { workspace = true }
+move-ethereum-abi = { path = "../move-ethereum-abi" }
```

Comment:
Could be made `{ workspace = true }`?

Reply:
Same with `move-prover-test-utils` etc below?

Reply:
For prover-test-utils and evm, the reasoning is the following:
- Prover test utils seems to be used only for prover. I do not know if we want to keep them internal or add as workspace? Or maybe it doesn't matter? What do you think?
- For EVM, I am not even sure we need this in Move repo, this is not really used anywhere, so kept as it hoping we just remove the whole thing😄

Reply:
> Prover test utils seems to be used only for prover

If you search for `prover-test-utils` in *.toml files, it shows that compiler-v2, linter, decompiler, etc, use it. It's name is perhaps outdated now, but it is used outside of the prover too.

Reply:
Indeed, missed that, updated to workspace👍